tests/test_data/pinned.json

```diff
@@ -10,7 +10,7 @@
   "delegations": [
     {
       "paths": ["*"],
-      "repositories": ["mainrepo", "director"]
+      "repositories": ["imagerepo", "director"]
```

Comment:
I feel that it should be `image_repo` everywhere.  "Image repository" is two words, and it's a bit harder to parse "imagerepo"